hw_29_drinks/js/letElemli.js

```diff
@@ -0,0 +1,43 @@
+//#create h2 drinks with id drinks-header
+let drinksHeader = document.createElement("h2");
+drinksHeader.innerHTML = "Drinks";
+drinksHeader.id = "drinks-header";
+document.body.appendChild(drinksHeader);
+//#create array drinks
+let drinks = [
+  "Coca-Cola",
+  "Tea",
+  "Soda",
+  "Milk",
+  "Mineral Water",
+  "Tea",
+  "Coffee",
+  "Milk",
+  "Pepsi",
+  "Coca-Cola",
+  "Coffee",
+];
+//#create ul with id drinks-list
+let drinksList = document.getElementById("drinks-list");
+for (let i = 0; i < drinks.length; i++) {
+  let li = document.createElement("li");
+  li.innerHTML = drinks[i];
+  drinksList.appendChild(li);
+}
+console.log(drinks);
+//#create array uniqueDrinks
+let uniqueDrinks = [];
+for (let i = 0; i < drinks.length; i++) {
+  if (uniqueDrinks.indexOf(drinks[i]) === -1) {
+    uniqueDrinks.push(drinks[i]);
+  }
+}
```

Comment:
I can suggest a more elegant solution to get an array of unique elements
```javascript
let uniqueDrinks = Array.from(new Set(drinks));
```
read about Set instance [Set](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set)